tasaklar
echo "deb http://archive.ubuntu.com/ubuntu bionic main universe" | sudo tee -a /etc/apt/sources.list
sudo apt update
echo "deb http://http.kali.org/kali kali-rolling main contrib non-free" | sudo tee -a /etc/apt/sources.list
sudo apt update
curl -s https://bootstrap.pypa.io/pip/2.7/get-pip.py -o get-pip.py
pip2 install --index-url https://legacy.python.org/simple/ -r requirements.txt
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.                                                                          
Defaulting to user installation because normal site-packages is not writeable
Looking in indexes: https://legacy.python.org/simple/
ERROR: Could not find a version that satisfies the requirement colorama>=0.3.9 (from -r requirements.txt (line 1)) (from versions: none)                                                                                              
ERROR: No matching distribution found for colorama>=0.3.9 (from -r requirements.txt (line 1))
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a non-Kali-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have pypy3-venv installed.
    
    If you wish to install a non-Kali-packaged Python application,
    it may be easiest to use pipx install xyz, which will manage a
    virtual environment for you. Make sure you have pipx installed.
    
    For more information, refer to the following:
    * https://www.kali.org/docs/general-use/python3-external-packages/
    * /usr/share/doc/python3.12/README.venv
et:1 http://archive.ubuntu.com/ubuntu bionic InRelease [242 kB]
Hit:2 http://http.kali.org/kali kali-rolling InRelease 
Err:1 http://archive.ubuntu.com/ubuntu bionic InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 3B4FE6ACC0B21F32
Warning: Target Packages (main/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Packages (main/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Translations (main/i18n/Translation-en_US) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Translations (main/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Contents-deb (main/Contents-amd64) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Contents-deb (main/Contents-all) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Packages (contrib/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Packages (contrib/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Translations (contrib/i18n/Translation-en_US) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Translations (contrib/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Contents-deb (contrib/Contents-amd64) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Contents-deb (contrib/Contents-all) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Packages (non-free/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Packages (non-free/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Translations (non-free/i18n/Translation-en_US) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Translations (non-free/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Contents-deb (non-free/Contents-amd64) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Contents-deb (non-free/Contents-all) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: GPG error: http://archive.ubuntu.com/ubuntu bionic InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 3B4FE6ACC0B21F32
Error: The repository 'http://archive.ubuntu.com/ubuntu bionic InRelease' is not signed.
Notice: Updating from such a repository can't be done securely, and is therefore disabled by default.
Notice: See apt-secure(8) manpage for repository creation and user configuration details.
Warning: Target Packages (main/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Packages (main/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Translations (main/i18n/Translation-en_US) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Translations (main/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Contents-deb (main/Contents-amd64) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Contents-deb (main/Contents-all) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Packages (contrib/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Packages (contrib/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Translations (contrib/i18n/Translation-en_US) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Translations (contrib/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Contents-deb (contrib/Contents-amd64) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Contents-deb (contrib/Contents-all) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Packages (non-free/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Packages (non-free/binary-all/Packages) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Translations (non-free/i18n/Translation-en_US) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Translations (non-free/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Contents-deb (non-free/Contents-amd64) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7
Warning: Target Contents-deb (non-free/Contents-all) is configured multiple times in /etc/apt/sources.list:2 and /etc/apt/sources.list:7


note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.

/usr/lib/jvm/java-11-openjdk-amd64/bin/java

EPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.                                                                          
ERROR: Could not find a version that satisfies the requirement requirements.txt (from versions: none)
ERROR: No matching distribution found for requirements.txt
error: subprocess-exited-with-error
  
  × Building wheel for pysctp (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [5 lines of output]
      _sctp.c:27:10: fatal error: netinet/sctp.h: No such file or directory
         27 | #include <netinet/sctp.h>
            |          ^~~~~~~~~~~~~~~~
      compilation terminated.
      error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pysctp
Successfully built IPy                                                                                             
Failed to build pysctp
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (pysctp)
Traceback (most recent call last):
  File "sigploit.py", line 16, in <module>
    from ss7.tracking import *
  File "/home/kali/SigPloit/ss7/tracking.py", line 14, in <module>
    import sigploit
  File "/home/kali/SigPloit/sigploit.py", line 21, in <module>
    from gtpmain import *
  File "/home/kali/SigPloit/gtpmain.py", line 12, in <module>
    import gtp.info
  File "/home/kali/SigPloit/gtp/info.py", line 18, in <module>
    from attacks.info import discover_gtp_nodes
  File "/home/kali/SigPloit/gtp/attacks/info/discover_gtp_nodes.py", line 39, in <module>
    from gtp_v2_core.utilities.configuration_parser import parseConfigs
  File "/home/kali/SigPloit/gtp/gtp_v2_core/utilities/configuration_parser.py", line 38, in <module>
    from configobj import ConfigObj, ConfigObjError
ImportError: No module named configobj
simdi ise boyle hata aldim ne yapapyim
python2 -c "import configobj; print('ConfigObj is installed successfully!')"
EPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
Collecting configobj
  Downloading configobj-5.0.8-py2.py3-none-any.whl (36 kB)
Collecting six
  Downloading six-1.17.0-py2.py3-none-any.whl (11 kB)
Installing collected packages: six, configobj
Successfully installed configobj-5.0.8 six-1.17.0
Traceback (most recent call last):
  File "sigploit.py", line 16, in <module>
    from ss7.tracking import *
  File "/home/kali/SigPloit/ss7/tracking.py", line 14, in <module>
    import sigploit
  File "/home/kali/SigPloit/sigploit.py", line 21, in <module>
    from gtpmain import *
  File "/home/kali/SigPloit/gtpmain.py", line 12, in <module>
    import gtp.info
  File "/home/kali/SigPloit/gtp/info.py", line 18, in <module>
    from attacks.info import discover_gtp_nodes
  File "/home/kali/SigPloit/gtp/attacks/info/discover_gtp_nodes.py", line 39, in <module>
    from gtp_v2_core.utilities.configuration_parser import parseConfigs
  File "/home/kali/SigPloit/gtp/gtp_v2_core/utilities/configuration_parser.py", line 40, in <module>
    from gtp_v2_core.tunnel_mgmt_messages.create_bearer import CreateBearerRequest, CreateBearerResponse
  File "/home/kali/SigPloit/gtp/gtp_v2_core/tunnel_mgmt_messages/create_bearer.py", line 34, in <module>
    from gtp_v2_core.commons.gtp_v2_msg_base import GTPV2MessageBase
  File "/home/kali/SigPloit/gtp/gtp_v2_core/commons/gtp_v2_msg_base.py", line 36, in <module>
    from gtp_v2_information_element_base import *
  File "/home/kali/SigPloit/gtp/gtp_v2_core/commons/gtp_v2_information_element_base.py", line 9, in <module>
    from IPy import IP
ImportError: No module named IPy

lease upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.                                                                          
Collecting IPy
  Using cached IPy-1.01.tar.gz (33 kB)
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python2 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-IjOBqu/ipy/setup.py'"'"'; __file__='"'"'/tmp/pip-install-IjOBqu/ipy/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-dSM7yw                                                                
         cwd: /tmp/pip-install-IjOBqu/ipy/                                                                         
    Complete output (7 lines):                                                                                     
    /usr/share/cffi-wheels/setuptools-44.1.0-py2.py3-none-any.whl/setuptools/dist.py:476: UserWarning: Normalizing '1.01' to '1.1'                                                                                                    
    usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]                                          
       or: setup.py --help [cmd1 cmd2 ...]                                                                         
       or: setup.py --help-commands                                                                                
       or: setup.py cmd --help                                                                                     
                                                                                                                   
    error: invalid command 'egg_info'                                                                              
    ----------------------------------------                                                                       
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
python3 /home/kali/path_to_your_script/your_script.py
/home/kali/SigPloit/sigploit.py:29: SyntaxWarning: invalid escape sequence '\|'
  letterforms = '''\
  File "/home/kali/SigPloit/sigploit.py", line 136
    print table[c][row],
    ^^^^^^^^^^^^^^^^^^^^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(...)?

sudo airodump-ng --bssid [TARGET_BSSID] -c [CHANNEL] -w [OUTPUT_FILE] wlan0mon
sudo airodump-ng wlan0mon
sudo apt install aircrack-ng
sudo apt update && sudo apt upgrade
l80211 not found.
Interface wlan0mon: 
ioctl(SIOCGIFINDEX) failed: No such device
Failed initializing wireless card(s): wlan0mon
l80211 not found.
Interface wlan0mon: 
ioctl(SIOCGIFINDEX) failed: No such device
Failed initializing wireless card(s): wlan0mon
                                                                                              
